packages/terra-table/tests/nightwatch/selectable-table/selectable-table-rows-spec.js

```diff
@@ -1,74 +0,0 @@
-/* eslint-disable no-unused-expressions */
```

Comment:
Why did we drop these?

Reply:
not dropped just moved to  packages/terra-table/nightwatch/selectable-table-rows-spec.js